spec/controllers/account/tenants_controller_spec.rb

```diff
@@ -5,18 +5,47 @@
     @user = create(:user, :super_admin)
     sign_in @user
   end
-  let!(:tenant) { FactoryBot.create(:tenant) }
-  render_views
-
-  it 'GET #index' do
-    get :index
-    expect(response).to have_http_status(200)
-    expect(response).to render_template(:index)
-    expect(response.body).to match(tenant.name)
+  let!(:tenant) { @user.tenant }
+
+  describe 'GET #index' do
+    context 'without render_views' do
+      it 'populates an array of tenants' do
+        get :index
+        expect(assigns(:tenants)).to eq([tenant])
+      end
+
+      it "renders the RSpec generated template" do
+        get :index
+        expect(response.body).to eq("")
+      end
+
+      it 'renders the :index view' do
+        get :index
+        expect(response).to have_http_status(200)
+        expect(response).to render_template(:index)
+      end
+    end
+
+    context 'with render_views' do
+      render_views
```

Comment:
render_views should be declared globally and used in all specs. I think it's already added. so could be removed. If not added - please add